src/clarity-angular/wizard/_wizard.clarity.scss

```diff
@@ -538,6 +538,10 @@
             min-height: 100%;
             height: auto;
             max-height: 100%;
+
+            .clr-wizard-stepnav {
```

Comment:
Do we want to do this everywhere for all wizards or does this just need to be for inline wizards?

Did we test this change to verify there's no unexpected issues with it?

Reply:
My understanding if the issue was that this was for inline wizards only. Added it for all wizards breaks our cssRegression tests for most wizardz ... they all get scroll bars as the `.clr-wizard-stepnav` pushes below the container. 

I'll update the description for browsers tested in.

Reply:
Cool. That's what I thought. Thanks for the clarification.